apim-lab/6-analyticsMonitoring/apimanagement-6-3-EventHub.md

```diff
@@ -52,9 +52,9 @@ To create an event hub within the namespace, follow these steps:
 
 1. Type a name for your event hub, then select **Create**.
        
-    The **partition count** setting allows you to parallelize consumption across many consumers. For more information, see [Partitions](event-hubs-scalability.md#partitions).
+    The **partition count** setting allows you to parallelize consumption across many consumers. For more information, see [Partitions](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-features#partitions).
 
-    The **message retention** setting specifies how long the Event Hubs service keeps data. For more information, see [Event retention](event-hubs-features.md#event-retention).
+    The **message retention** setting specifies how long the Event Hubs service keeps data. For more information, see [Event retention](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-features#event-publishers).

```

Comment:
Please update the link to https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-features#event-retention